webrtc.html

```diff
@@ -7520,6 +7557,12 @@ <h3>Simulcast functionality</h3>
         implications of this model is that the <code>addTrack</code> method cannot provide simulcast
         functionality since it does not take <code>sendEncodings</code> as an argument, and therefore cannot
         configure an <code><a>RTCRtpTransceiver</a></code> to send simulcast.</p>
+        <p>The answerer MUST NOT set the <a>simulcast envelope</a> directly. Upon calling the
```

Comment:
I apologize. Now that you've added normative steps in the algorithm itself (which I like much better!), this section should probably no longer use normative language, in order to not compete. What you had here before I meddled was probably fine, and gelled better with the paragraph directly above and below it. Can you put it back? Maybe with a bridge like:

"Another implication is that the answerer cannot set the <a>simulcast envelope</a> directly."

Some specs might even mark this whole section as *non-normative*, as it appears to merely summarize behavior dictated elsewhere, although I don't know if @aboba would go for that, in case there are other important bits mentioned only here?